examples/dummy-iiostream.c

```diff
@@ -315,27 +322,43 @@ int main (int argc, char **argv)
 
 		// Print timestamp delta in ms
 		if (has_ts)
-			for (p_dat = iio_buffer_first(rxbuf, channels[channel_count-1]); p_dat < p_end; p_dat += p_inc) {
+			for (p_dat = iio_buffer_first(
+					     rxbuf,
+					     channels[channel_count - 1]);
+			     p_dat < p_end; p_dat += p_inc) {
 				now_ts = (((int64_t *)p_dat)[0]);
-				printf("[%04" PRId64 "] ", last_ts > 0 ? (now_ts - last_ts)/1000/1000 : 0);
+				printf("[%04" PRId64 "] ",
+				       last_ts > 0 ? (now_ts - last_ts) / 1000 /
+								       1000 :
+						     0);
 				last_ts = now_ts;
 			}
 
 		// Print each captured sample
-		switch (buffer_read_method)
-		{
+		switch (buffer_read_method) {
 		case BUFFER_POINTER:
 			for (int i = 0; i < channel_count; ++i) {
-				const struct iio_data_format *fmt = iio_channel_get_data_format(channels[i]);
-				unsigned int repeat = has_repeat ? fmt->repeat : 1;
+				const struct iio_data_format *fmt =
+						iio_channel_get_data_format(
+								channels[i]);
+				unsigned int repeat =
+						has_repeat ? fmt->repeat : 1;
 
 				printf("%s ", iio_channel_get_id(channels[i]));
-				for (p_dat = iio_buffer_first(rxbuf, channels[i]); p_dat < p_end; p_dat += p_inc) {
+				for (p_dat = iio_buffer_first(rxbuf,
+							      channels[i]);
+				     p_dat < p_end; p_dat += p_inc) {
 					for (int j = 0; j < repeat; ++j) {
-						if (fmt->length/8 == sizeof(int16_t))
-							printf("%" PRIi16 " ", ((int16_t *)p_dat)[j]);
-						else if (fmt->length/8 == sizeof(int64_t))
-							printf("%" PRId64 " ", ((int64_t *)p_dat)[j]);
+						if (fmt->length / 8 ==
```

Comment:
this part looks pretty bad when trying to avoid the 80 col width limit;
we can leave it like this for now, but we may want to move this to it's own function with early-exits

Reply:
I'm Ok moving things to more than 80 col if that was the suggestion?

Reply:
either is fine; we can leave a higher-than-80-col-width;
or we can leave it like this;
though, the way it is now [with the tool applied], is kind of bad;

we can also clean it up later;